src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,19 @@
+package com.example;
+
+/**
+ * A customer is a person who visits a shop for buying goods.
+ * The customer can have a loyaltyCard that has a specific number.
```

Comment:
The 2nd paragraph must start after an empty line and have `<p>` before the first letter of the text. This is so because it's a paragraph.

Reply:
> `loyaltyCard`

This looks like a name in a problem domain, not a piece of code.

If this is a name in the code it should be either `{@code LoyaltyCard}` or `{@link LoyaltyCard}`. If not, it should go just plain text.

Reply:
Another thing, details of the `LoyaltyCard` type should be described in that class. It could be a number, a QR code, a finger print, or retina scan. Being inside the customer, we don't care. We get an instance of `LoyaltyCard` and we use its API.  Internals must be encapsulated (hidden) as much as possible. This concerns the documentation too.

Reply:
This group of comments was not addressed. Please do.

Reply:
@elena-yevs, the second sentence in Javadoc must start with `<p>`. Please review Javadocs in Guava to understand what it looks like.